front/src/common/Pathfinding/pathfinding.scss

```diff
@@ -0,0 +1,70 @@
+.pathfinding-main-container {
+  margin-left: 1.5rem;
+
+  .pathfinding-subsection {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .pathfinding-done {
+    @extend .pathfinding-subsection;
+    
+    .title {
+      color: green;
+    }
+
+    .details {
+      font-weight: 400;
+    }
+  }
+
+  .pathfinding-error {
+    @extend .pathfinding-subsection;
+    color: red;
+  }
+
+  .missing-params {
+    @extend .pathfinding-subsection;
+    color: #ffb612;
```

Comment:
please use Bootstrap's color with var(--orange) for example

Reply:
done